fipy/meshes/factoryMeshes.py

```diff
@@ -49,8 +49,10 @@ def Grid3D(dx=1., dy=1., dz=1.,
            overlap=2, communicator=parallelComm):
 
     r""" Factory function to select between UniformGrid3D and
-    NonUniformGrid3D.  If `Lx` is specified the length of the domain
-    is always `Lx` regardless of `dx`.
+    NonUniformGrid3D. If `L{xyz}` is specified, the length of the domain is
+    always `L{xyx}` regardless of `d{xyz}`, unless `d{xyz}` is a list of
```

Comment:
It would perhaps be more succinct (and accurate) to skip the commas and amend the documentation to "you'll get whatever mesh we give you and you'll choose to like it"